package.json

```diff
@@ -35,11 +35,18 @@
   },
   "devDependencies": {
     "@mapbox/node-pre-gyp": "1.0.6",
+    "@wdio/cli": "8.22.1",
```

Comment:
Can we avoid using @wdio ?

Reply:
resolved, we're moving forward with the `WebdriverIO`